ui/src/app/views/workspace/component.ts

```diff
@@ -204,6 +195,19 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
     this.bugReportOpen = true;
   }
 
+  workspaceClustersBillingProjectId(): string {
```

Comment:
nit: I would not recommend naming methods/variables with a possessive, I think it's a bit ambiguous with the plural. I would just name this workspaceClusterBillingProjectId